hack/cut-release.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+# Copyright The Enterprise Contract Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+if [[ $CURRENT_BRANCH != "main" ]]; then
+  echo "Expecting to be in main branch!"
+  exit 1
+fi
+
+RELEASE_NAME=${1:-""}
+
+if [[ $RELEASE_NAME == "" ]]; then
+  echo "Please provide a release name, e.g. v0.1-preview, or v1.1"
+  exit 1
+fi
+
+BRANCH_NAME="redhat-${RELEASE_NAME}"
+RHTAP_APPLICATION_NAME="ec-${RELEASE_NAME/./}"
+
+echo Release name: $RELEASE_NAME
+echo Release branch name: $BRANCH_NAME
+echo RHTAP application name: $RHTAP_APPLICATION_NAME
+
+echo "Creating branch"
+git checkout -b $BRANCH_NAME main
```

Comment:
Do we need to make sure we have the latest upstream rev?  `git fetch upstream` and `git checkout -b $BRANCH_NAME upstream/main`?